microreactors/gcmr/core/Neutronics/tests

```diff
@@ -0,0 +1,30 @@
+[Tests]
+  parallel_scheduling = True
+  # Steady state
+  # Griffin tests
+  [griffin_alone_syntax]
```

Comment:
why _alone? are there other inputs to come?

Reply:
Yes, we are planning to add Multiphysics later.

Reply:
Yes, we are planning to add Multiphysics later.